res/css/structures/_SpacePanel.scss

```diff
@@ -154,9 +159,8 @@ $activeBorderColor: $primary-content;
             align-items: center;
             border-radius: 12px;
             padding: 4px;
-            width: calc(100% - 32px);
+            width: 32px;
```

Comment:
This doesn't actually have the effect of setting the width to 32px, instead what it's doing is lowering the min-width of the element to allow it to collapse. So could you please replace this with `min-width: 0;` to be more clear that that is what this accomplishes?

Once you do that, you may have to set `min-width: 0;` on `.mx_SpaceTreeLevel` as well, but it should overall be the more correct approach.

Reply:
Noted